packages/rocketchat-channel-settings/client/views/channelSettings.coffee

```diff
@@ -106,4 +98,13 @@ Template.channelSettings.onCreated ->
 								return toastr.error TAPi18n.__(err.reason, err.details.roomType)
 							return toastr.error TAPi18n.__(err.reason)
 						toastr.success TAPi18n.__ 'Room_type_changed_successfully'
+			when 'archivationState'
+				if @$('input[name=archivationState]:checked').val() is 'true'
+					Meteor.call 'archiveRoom', @data?.rid, (err, results) ->
```

Comment:
@graywolf336 sorry about that, I will try to explain better on the next time, fell free to correct me, I'm learning english yet :smile: 

@matthias-brun thanks :smiley: